Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/event/model/Document.java

```diff
@@ -419,7 +421,7 @@ public void setContentType(String contentType) {
      * @return the documentType
      */
     public String getDocumentType() {
-        return documentType;
+        return (documentType == null) ? documentClassCode : documentType;
```

Comment:
Confused about the naming here.  We call it "getDocumentType", but that's not necessarily what it does.  Same for "getDocumentId" above.  Should this be renamed, or the functionality split?